app/uk/gov/hmrc/vatsubscription/connectors/GetVatCustomerInformationConnector.scala

```diff
@@ -18,13 +18,13 @@ package uk.gov.hmrc.vatsubscription.connectors
 
 import javax.inject.{Inject, Singleton}
 import play.api.Logger
-import play.api.http.Status.{BAD_REQUEST, FORBIDDEN, NOT_FOUND, OK, PRECONDITION_FAILED, INTERNAL_SERVER_ERROR}
+import play.api.http.Status.{BAD_REQUEST, FORBIDDEN, INTERNAL_SERVER_ERROR, NOT_FOUND, OK, PRECONDITION_FAILED}
 import play.api.libs.json.{JsSuccess, Json, Writes}
 import uk.gov.hmrc.http.logging.Authorization
 import uk.gov.hmrc.http.{HeaderCarrier, HttpReads, HttpResponse}
 import uk.gov.hmrc.play.bootstrap.http.HttpClient
 import uk.gov.hmrc.vatsubscription.config.AppConfig
-import uk.gov.hmrc.vatsubscription.config.featureSwitch.{Api1363R10, Api1363R8}
+import uk.gov.hmrc.vatsubscription.config.featureSwitch.Api1363Latest
```

Comment:
This import is unused now that we don't have to match a version and accept latest as the default.